application_authentication_handlers_test.go

```diff
@@ -217,6 +220,71 @@ func TestApplicationAuthenticationGet(t *testing.T) {
 	if out.ApplicationID != "1" {
 		t.Error("ghosts infected the return")
 	}
+
+	// Check the tenancy of returned app auth
+	for _, appAuth := range fixtures.TestApplicationAuthenticationData {
+		if fmt.Sprintf("%d", appAuth.ID) == out.ApplicationID {
```

Comment:
I think that here it should be `out.ID`, right?

Reply:
that is really awesome catch !!!! sharp eye ... I changed the app auth id that is used for this test to have better test data (previous app auth id = 1 was relation between app id = 1 and auth id = 1 and that is the reason why it worked with` out.ApplicationID`)